setup.py

```diff
@@ -19,17 +19,17 @@
 
 tests_require = [
     'SQLAlchemy-Continuum>=1.3.6',
-    'SQLAlchemy<1.4.0,>=1.2.18',
-    'SQLAlchemy-Utils<0.36,>=0.33.1',
-    'invenio-indexer>=1.2.1',
+    'SQLAlchemy>=1.2.18,<1.5.0',
```

Comment:
I think all these dependencies should be removed instead and fully rely on InvenioDB

Reply:
I agree, but we can only remove. Adding invenio-db here will have no effect, since it should be installed as extra in tests.